### PR TITLE
feat: kube 1.22 support

### PR DESCRIPTION
**Pull request by dnplkndll**

### CODE REVIEW

**Comment by clglavan** [comment]
Hello, any updates on this?

**Comment by hors** [comment]
> Hello, any updates on this?

Thanks for PR, we will check it soon.  We do not recommend  this chart for production environment. We use it only for the testing need. 

JFYI we have official helm chart for PMM server. Please have a look.
https://github.com/percona/percona-helm-charts/tree/main/charts/pmm

File: helm/pmm-server/templates/statefulset.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
           ports:
             - name: https
               protocol: TCP
-              containerPort: 443
+              containerPort: 80
```
Comment (tplavcic) [inline helm/pmm-server/templates/statefulset.yaml:43]: Name is left `https` and port is `80`. Also could you explain why this change is needed?

**Review by tplavcic — CHANGES_REQUESTED** [review]
Please bump the chart version.

File: helm/pmm-server/templates/statefulset.yaml (hunk 1 above)
Comment (dnplkndll) [reply helm/pmm-server/templates/statefulset.yaml:43]: the name should be http to match 80. I guess the only part needed here was the 
```
apiVersion: extensions/v1beta1
```
the rest just maps to 80 singe the acme cert-manager would provide the ssl. not needed in the stateful deployment unless used for something else. 
We have since migrated to the chart recommended an so I no longer have the one in use.

**Comment by cap1984** [comment]
Hey @dnplkndll 

We are to drop this chart support. Please use the official one https://github.com/percona/percona-helm-charts/tree/main/charts/pmm
Thanks
